operator-common/src/main/java/io/strimzi/operator/common/operator/resource/ServiceOperator.java

```diff
@@ -87,9 +89,9 @@ protected Pattern ignorablePaths() {
                 patchDualStackNetworking(current, desired);
             }
 
-            return super.internalPatch(namespace, name, current, desired);
+            return super.internalPatch(reconciliation, namespace, name, current, desired);
         } catch (Exception e) {
-            log.error("Caught exception while patching {} {} in namespace {}", resourceKind, name, namespace, e);
+            reconciliationLogger.errorCr(reconciliation, "Caught exception while patching {} {} in namespace {}", resourceKind, name, namespace, e);
```

Comment:
Same as above ... should this be just LOGGER? I won't repeat this comment again.